Tools/nib2cib/NSBox.j

```diff
@@ -71,9 +71,8 @@
         }
 
         _title         = [[aCoder decodeObjectForKey:@"NSTitleCell"] objectValue] || @"";
-        _titlePosition = [aCoder decodeObjectForKey:@"NSTitlePosition"];
 
-        if (_titlePosition === undefined)
+        if (_titlePosition === undefined && _boxType !== CPBoxSeparator)
```

Comment:
Hi enquora,
Is there a reason why titlePosition should not be decoded ? if you need to test for the presence of a key in the CPKeyedUnarchiver you can use ```-containsValueForKey``` and then set the value, including a default value.